.github/workflows/ci-python.yml

```
@@ -127,14 +127,24 @@ jobs:
         if: startsWith(matrix.os, 'windows-')
         shell: bash
         run: |
+          set -eux
           choco install -y --limit-output winfsp --version=${{ matrix.winfsp-version }}
           mkdir winfsp-test
           curl -L https://github.com/billziss-gh/winfsp/releases/download/v1.8/winfsp-tests-1.8.20304.zip -o winfsp-test/winfsp-tests.zip
           unzip winfsp-test/winfsp-tests.zip -d winfsp-test
```

Use The same path everywhere & use the temp directory instead.

```suggestion
          mkdir -vp ${{ runner.temp }}/winfsp-test
          curl -L https://github.com/billziss-gh/winfsp/releases/download/v1.8/winfsp-tests-1.8.20304.zip -o ${{ runner.temp }}/winfsp-test/winfsp-tests.zip
          unzip ${{ runner.temp }}/winfsp-test/winfsp-tests.zip -d ${{ runner.temp }}/winfsp-test
```

---

is `runner.temp` really persistent between jobs? is it really worth making this change?

---

`runner.temp` goal is not to be persistent between job (the directory is cleared).
Using `github.workspace` or `runner.temp` is the same, the installed `winfst-tests` isn't keep

---

`github.workspace` is persistent, otherwise our winfsp-tests wouldn't run. 
The whole point of this is task to make the `winfsp-tests` executable available for the futures tasks, in particular the mounpoint tests.